contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -4451,6 +4451,37 @@ def testAssetPathSlashes ( self ) :
 		self.assertNotIn( "\\", xform.readAttribute( "render:testAsset", 0 ).value )
 		self.assertTrue( pathlib.Path( xform.readAttribute( "render:testAsset", 0 ).value ).is_file() )
 
+
+	@unittest.skipIf( os.environ.get("GAFFERSCENE_INSTANCER_EXPLICITABSOLUTEPATHS") == "1" or os.environ.get("GAFFERUSD_POINTINSTANCER_RELATIVEPROTOTYPES") == "0", "GAFFERSCENE_INSTANCER_EXPLICITABSOLUTEPATHS or GAFFERUSD_POINTINSTANCER_RELATIVEPROTOTYPES is set, running alternate test" )
```

Comment:
> Gaffer has tests which set env vars and then relaunch themselves in a subprocess, but I wasn't sure how to do that in the Cortex test setup.

What have you tried so far? I would have thought it wasn't too hard to do something similar with `subprocess` and `__file__`. Or maybe `multiprocessing` if you're feeling fancy.